app/controllers/funds_controller.rb

```diff
@@ -0,0 +1,9 @@
+class FundsController < ApplicationController
+  def index
+    funds = Rails.cache.fetch("all_funds", expires_in: 24.hours) { Fund.all }
```

Comment:
Nice! 🔥 
_We may want to make emphasis on this on the README_ if futre devs encounter the case that the update to funds "is not working, as it is not being updated" anyone can see in plain sight that this is expected to be cached 24h.

_Or this will behave differently when the records change?_

Reply:
yeah, we can expire the cache after a fund is created/updated. I'll add this.

Reply:
changes were made